gunicorn/config.py

```diff
@@ -57,13 +57,13 @@ def __str__(self):
         for k in sorted(self.settings):
             v = self.settings[k].value
             if callable(v):
-                v = "<{}()>".format(v.__qualname__)
-            lines.append("{k:{kmax}} = {v}".format(k=k, v=v, kmax=kmax))
+                v = f"<{v.__qualname__}()>"
+            lines.append(f"{k:{kmax}} = {v}")
```

Comment:
Missing escaping here, I think (so probably some additional test coverage required)

Reply:
It turns out that this was not an escaping issue after all.

- Legacy [Python string formatting](https://docs.python.org/3.6/library/string.html?highlight=str#format-examples) using the string `%` operator _does_ retain curly braces
  - For example, `"{%s}" % "hi"` evaluates to `"{hi}"`
- `str.format` string formatting uses curly-braces to delimit positional and/or named arguments
  - For example, `"{}".format("hi")` evaluates to `"hi"`

The code referenced by this comment was mistaken for legacy-string formatting (which would require escaping of curly braces) but is in fact `str.format` based (so no escaping of curly braces is required).

Even so: the use of `kmax` as a fill operator seemed not-entirely-intuitive (although very neat), so some test coverage for that behaviour has been added in 719a4b1b5023161187bc73c695cf26d7cfaaa469 (with demonstration that it does catch errors in 9f0f9a2917e0aadb01cd92308ce456be58f67bed -- subsequently reverted by 7bc470853c7338ade6c9f84eeed5c4d80c96d56c).